### DISCUSSION TITLE
Support for Compose Animation

### DISCUSSION THREAD

**Discussion by basimsherif** [cashapp/paparazzi#788] [Q&A] [answered]
How can we generate snapshots for a Composable with Animation? Looks like Paparazzi is not waiting for Animation to start even with putting some delay() and LaunchedEffect in test. This makes Composable hidden in the screenshot taken.

**Comment by geoff-powell** [comment, marked as answer]
You can capture animations by using `paparazzi.gif` to generate an APNG of said composable animation.

**Comment by vitorhugods** [comment]
I couldn't find a `gif` API for Composables in `2.0.0-alpha02`, so here's what I did:
```kotlin
val paparazzi = // Initialize Paparazzi...


val context = RenderAction.getCurrentContext()
val hostView = ComposeView(context).apply {
    layoutParams = ViewGroup.LayoutParams(ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT)
}
hostView.setContent { MyCoolComposable() }
paparazzi.gif(hostView)
```
